tools/ttmlir-opt/CMakeLists.txt

```diff
@@ -1,9 +1,17 @@
 get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
 get_property(conversion_libs GLOBAL PROPERTY MLIR_CONVERSION_LIBS)
-set(LIBS ${dialect_libs} ${conversion_libs} MLIROptLib MLIRTargetCpp TTMLIRStatic)
+set(LIBS ${dialect_libs} ${conversion_libs} StablehloBase TTNNTargetFlatbuffer MLIRTTIRDialect TTMLIRConversions MLIRTransformUtils MLIROptLib MLIRTargetCpp TTMLIRStatic CheckOps ChloCAPI ChloOps InterpreterOps StablehloAssemblyFormat StablehloBase StablehloBroadcastUtils StablehloCAPI StablehloLinalgTransforms StablehloOps StablehloPasses StablehloPortableApi StablehloRegister StablehloSerialization StablehloTOSATransforms StablehloTestUtils StablehloTypeInference Version VhloCAPI VhloOps VhloTypes)
```

Comment:
All of these `TTNNTargetFlatbuffer MLIRTTIRDialect TTMLIRConversions MLIRTransformUtils` should be captured by `TTMLIRStatic`.

We never needed to link these ones before `MLIRTransformUtils MLIROptLib MLIRTargetCpp`, MLIR should automatically pull those in via the variables at the start.

All of the stablehlo libs, shouldn't those go underneath the guard you have below that links `TTMLIRStableHLOToTTIR`?